packages/alice/lib/helper/alice_save_helper.dart

```diff
@@ -74,145 +79,138 @@ class AliceSaveHelper {
         );
         return '';
       }
-      final isAndroid = Platform.isAndroid;
-      final isIOS = Platform.isIOS;
-
-      Directory? externalDir;
-      if (isAndroid) {
-        externalDir = await getExternalStorageDirectory();
-      } else if (isIOS) {
-        externalDir = await getApplicationDocumentsDirectory();
-      } else {
-        externalDir = await getApplicationCacheDirectory();
-      }
+
+      final Directory? externalDir = switch (Platform.operatingSystem) {
+        "android" => await getExternalStorageDirectory(),
```

Comment:
I think `android` and `ios` could be moved to some consts definition in top of the class.

Reply:
Well yes, but then the 3-way `switch` statement won't work. 🤷

These are already constants in their own right, since

```dart
/// Whether the operating system is a version of
/// [Android](https://en.wikipedia.org/wiki/Android_%28operating_system%29).
@pragma("vm:platform-const")
static final bool isAndroid = (operatingSystem == "android");

/// Whether the operating system is a version of
/// [iOS](https://en.wikipedia.org/wiki/IOS).
@pragma("vm:platform-const")
static final bool isIOS = (operatingSystem == "ios");
```

Reply:
Done